website/pages/blog/announcing-self-hosted-graphql-hive.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Announcing self-hosted GraphQL Hive
+tags: [graphql, graphql-hive]
+authors: [laurin]
+date: 2022-10-04
+description: Host your own GraphQL Hive instance.
+image: /blog-assets/announcing-self-hosted-graphql-hive/cover.png
+thumbnail: /blog-assets/announcing-self-hosted-graphql-hive/thumbnail.png
+---
+
+It has been 5 months since [we first announced GraphQL Hive](/blog/graphql-hive-preview) and it has been a wild ride since then!
```

Comment:
`Mar 26th 2021`

Reply:
Maybe we should link to https://4e7b143f.guild-dev-website.pages.dev/blog/announcing-graphql-hive-release? It's when we went public and mention the preview program (/blog/graphql-hive-preview)?